substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVSymbolSubrecord.java

```diff
@@ -295,7 +378,7 @@ protected int computeContents(byte[] buffer, int initialPos) {
             pos = CVUtil.putInt(debugStart, buffer, pos);
             pos = CVUtil.putInt(debugEnd, buffer, pos);
             pos = CVUtil.putInt(typeIndex, buffer, pos);
-            pos = cvDebugInfo.getCVSymbolSection().markRelocationSite(buffer, pos, symbolName);
+            pos = cvDebugInfo.getCVSymbolSection().markRelocationSite(buffer, pos, symbolName, offset);
```

Comment:
Not sure what the intention was here, but adding a non-zero addend to a relocation that already calculates the correct value is wrong. Please revert this change.

Reply:
I have removed the offset parameter from this code.  To me, it should stay, because this is a generic implementation of a S_GPROC32 record.